grafana/values.yaml

```diff
@@ -0,0 +1,26 @@
+clusterName: "" # @schema description: Name of the cluster to configure the platform Grafana for. This value is required.
+
+baseDomain: "" ## @schema description: Base domain for cluster ingresses (e.g. dev-core-platform.dev.czi.team). This value is required.
+
+grafanaName: "grafana" # @schema description: Name of the Grafana instance to create.
+
+replicas: 1 # @schema description: Number of Grafana replicas to create. When greater than 1, database persistence is required (not supported yet), as well as session affinity. 
+
+secretStoreRef: "aws-secretsmanager" # @schema description: Name of the secret store to use for external secrets.
+
+roleAttributePath: "contains(groups[*], 'team-central-czi-admin') && 'Admin' || contains(groups[*], 'team-pep-dev-core-platform-argus-obs.Admin') && 'Admin' || 'Viewer'" # @schema description: roleAttributePath JMESPath expression to use to determine the role of the user. See https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/generic-oauth/ .
+
+grafanaAnnotations: {} # @schema description: Annotations to add to the Grafana instance.
```

Comment:
Should this be enforced to be `map[string]string`?

Reply:
If so, I think you would just change this to `grafanaAnnotations: {} # @schema description: Annotations to add to the Grafana instance.; patternProperties: { "^.*$": { "type": "string" } }`